LayoutTests/TestExpectations

```diff
@@ -6227,6 +6227,7 @@ imported/w3c/web-platform-tests/html/browsers/the-window-object/window-open-noop
 
 # CSS nesting unsupported yet
 imported/w3c/web-platform-tests/css/css-nesting/nest-containing-forgiving.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/css-nesting/nesting-basic.html [ ImageOnlyFailure ]
```

Comment:
do we know why it now fails?

Reply:
Yes, top-level nesting is still unsupported.